frame/session/src/lib.rs

```diff
@@ -125,6 +125,19 @@ pub mod historical;
 
 mod default_weights;
 
+pub trait ValidatorIdentification<AccountId> {
```

Comment:
I don't understand why this trait exists in the first place. Also there is no docs.